jenkinsbuilder/weekly-scan.py

```diff
@@ -86,9 +91,23 @@
                 "weekly": True,
                 "logs_dir": LOGS_DIR,
                 "test_tag": test_tag,
-                "job_name": job_id
+                "job_name": job_id,
+                "uuid": job_uuid
             }
 
             job = bs.put(json.dumps(data))
+
+            # Initializing Database entries
+            project, created = Project.objects.get_or_create(name=project_name)
+            build = Build.objects.create(uuid=job_uuid, project=project,
+                                         status='queued',
+                                         start_time=timezone.now(),
+                                         weekly_scan=True
+                                         )
+            scan_phase, created = BuildPhase.objects.get_or_create(
+                build=build, phase='scan')
+            scan_phase.status = 'queued'
```

Comment:
I think we already set `status=queued` on line number 183 of this file ? We dont need to re-set it to same value unless changed. Just remove line number 109 ?